### PR TITLE
Move enums to new package

### PR DESCRIPTION
**Pull request by jdahmubed**

### CODE REVIEW

File: src/org/oidc/service/base/enums/Algorithm.java [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package enums;
+package org.oidc.service.base.enums;
```
Comment (manu-sinha) [inline src/org/oidc/service/base/enums/Algorithm.java]: Like I mentioned yesterday , lot of the enums would be shared across. For example, Algorithm enum. Let's keep this in org.oidc.common package.

File: src/org/oidc/service/base/enums/ClientAuthenticationMethod.java [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package enums;
+package org.oidc.service.base.enums;
```
Comment (manu-sinha) [inline src/org/oidc/service/base/enums/ClientAuthenticationMethod.java]: Like I mentioned yesterday , lot of the enums would be shared across like this one. Let's keep this in org.oidc.common package.

File: src/org/oidc/service/base/enums/SerializationType.java [hunk 3]
```diff
@@ -1,4 +1,4 @@
-package enums;
+package org.oidc.service.base.enums;
```
Comment (manu-sinha) [inline src/org/oidc/service/base/enums/SerializationType.java]: Like I mentioned yesterday , lot of the enums would be shared across like this one. Let's keep this in org.oidc.common package.

File: src/org/oidc/service/base/enums/HttpMethod.java [hunk 4]
```diff
@@ -1,4 +1,4 @@
-package enums;
+package org.oidc.service.base.enums;
```
Comment (manu-sinha) [inline src/org/oidc/service/base/enums/HttpMethod.java]: Like I mentioned yesterday , lot of the enums would be shared across like this one. Let's keep this in org.oidc.common package.

File: src/org/oidc/service/base/enums/KeyType.java [hunk 5]
```diff
@@ -1,4 +1,4 @@
-package enums;
+package org.oidc.service.base.enums;
```
Comment (manu-sinha) [inline src/org/oidc/service/base/enums/KeyType.java]: Like I mentioned yesterday , lot of the enums would be shared across like this one. Let's keep this in org.oidc.common package.

File: src/org/oidc/service/base/enums/MessageType.java [hunk 6]
```diff
@@ -1,4 +1,4 @@
-package enums;
+package org.oidc.service.base.enums;
```
Comment (manu-sinha) [inline src/org/oidc/service/base/enums/MessageType.java]: Like I mentioned yesterday , lot of the enums would be shared across like this one. Let's keep this in org.oidc.common package.

**Review by manu-sinha — CHANGES_REQUESTED** [review]
Some of the enums would be shared across. Such enums should be in org.oidc.common package. If there are enums that are specific to services, those can be in org.oidc.service.base package. We don't need package to be org.oidc.service.base.enum. It can just be org.oidc.service.base. I see that you are having difficulty in deciding common enums vs service specific enums, hence lets keep all the enums in org.oidc.common for now.

File: src/org/oidc/service/base/enums/Algorithm.java (hunk 1 above)
Comment (jdahmubed) [reply src/org/oidc/service/base/enums/Algorithm.java]: moved all enums to org.oidc.common

**Review by manu-sinha — APPROVED** [review]
Looks good. Thank you!

**Pull request merged**